src/frontend-samples/classifier-sample/ClassifierWidget.tsx

```diff
@@ -0,0 +1,178 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React, { useCallback, useEffect } from "react";
+import { useActiveIModelConnection, useActiveViewport } from "@bentley/ui-framework";
+import { SpatialClassificationProps } from "@bentley/imodeljs-common";
+import { AbstractWidgetProps, StagePanelLocation, StagePanelSection, UiItemsProvider, WidgetState } from "@bentley/ui-abstract";
+import { KeySet } from "@bentley/presentation-common";
+import { Input, Select } from "@bentley/ui-core";
+import { IModelApp } from "@bentley/imodeljs-frontend";
+import { ISelectionProvider, SelectionChangeEventArgs } from "@bentley/presentation-frontend";
+import ClassifierApi from "./ClassifierApi";
+import { ClassifierProperties } from "./ClassifierProperties";
+import "./Classifier.scss";
+
+const ClassifierWidget: React.FunctionComponent = () => {
+  const _insideDisplayEntries: { [key: string]: string } = {};
+  _insideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.ElementColor]] = "Element Color";
+  _insideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.Off]] = "Off";
+  _insideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.On]] = "On";
+  _insideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.Dimmed]] = "Dimmed";
+  _insideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.Hilite]] = "Hilite";
+
+  const _outsideDisplayEntries: { [key: string]: string } = {};
+  _outsideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.Off]] = "Off";
+  _outsideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.On]] = "On";
+  _outsideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.Dimmed]] = "Dimmed";
+
+  const iModelConnection = useActiveIModelConnection();
+  const viewport = useActiveViewport();
+  const [initalized, setInitalized] = React.useState<boolean>(false);
+  const [classifiersState, setClassifiersState] = React.useState<{ [key: string]: string }>({});
+  const [classifierState, setClassifierState] = React.useState<string>();
```

Comment:
Maybe you can find better names. `classifiersState` and `classifierState` are a little too similar

Reply:
Renamed to 'classifiers' and 'currentClassifier' respectively.